SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -27,6 +27,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bEnableHandover(true)
 	, MaxNetCullDistanceSquared(900000000.0f) // Set to twice the default Actor NetCullDistanceSquared (300m)
 	, QueuedIncomingRPCWaitTime(1.0f)
+	, QueuedOutgoingRPCWaitTime(10.0f)
```

Comment:
This seems high - any reason it's set to that?

Reply:
Was my suggestion but yeah, probably much higher than needed. Could make it 1 second to match?

Reply:
Changed to 1 second